app/helpers/content_header_helper.rb

```diff
@@ -0,0 +1,211 @@
+module ContentHeaderHelper
+  class AbstractContent
+    include ActionView::Helpers
+    attr_accessor :output_buffer
+
+    delegate :key_for, to: OpenProject::AccessKeys
+  end
+
+  class ContentHeader < AbstractContent
+    def initialize(title:, subtitle: '', helper: nil, scrollable: false, &_block)
+      @title = title
+      @subtitle = subtitle
+      @helper = helper
+      @scrollable = scrollable
+    end
+
+    def toolbar(&block)
+      content_tag :div, class: 'toolbar-container' do
+        class_name = %w(toolbar)
+        class_name += %w(-scrollable) if @scrollable
+        header = content_tag :div, class: class_name, role: 'navigation' do
+          dom_title(@title) + content_tag(:ul, items(&block), class: 'toolbar-items', role: 'menubar')
+        end
+        next header if @subtitle.blank?
+        capture do
+          concat header
+          concat content_tag(:p, @subtitle, class: 'subtitle')
+        end
+      end
+    end
+
+    protected
+
+    def items(&block)
+      return @helper.capture(ContentToolbar.new, &block) if block_given?
+      ''
```

Comment:
Yeah, but it is 2 lines instead of 5. 

I could change it to:

```ruby
return '' unless block_given?
@helper.capture(ContentToolbar.new, &block)
```